recipes/gravelamps/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "gravelamps" %}
+{% set version = "2.3.1rc14" %}
```

Comment:
Your next release should be a stable release. If you need RCs in the future, please use a pre-release label.

I assume that you have been using this rc to make changes specifically for conda-forge.

https://conda-forge.org/docs/maintainer/knowledge_base.html#pre-release-builds